Apps/Contoso.WPF.Demo/MainWindow.xaml.cs

```diff
@@ -52,6 +53,12 @@ public MainWindow()
             textAttachments = Settings.Default.TextErrorAttachments;
             TextAttachmentTextBox.Text = textAttachments;
             FileAttachmentLabel.Content = fileAttachments;
+
+            if (!string.IsNullOrEmpty(Settings.Default.UserId))
```

Comment:
lets keep it in same place in all apps (after country code)

Reply:
I tried to do that. But seems like other Apps don't have CountryCode initialized in the MainWindow().

Reply:
Moved it down.